Slugbotics/ARM
# Added file logging system.

Added a system to log print messages to a file as well as the console output if a config option is enabled.

TO TEST:

Enable the config option "write_logs"

Run the main program

See if a directory called Logs is created, and that the file it creates contains the expected program output

If it does, then this looks good!
